Libplanet/Net/Swarm.cs

```diff
@@ -1298,17 +1387,11 @@ private void BroadcastTxIds(IEnumerable<TxId> txIds)
             BlockHashes message,
             CancellationToken cancellationToken = default(CancellationToken))
         {
-            if (!(message.Sender is Address from))
-            {
-                throw new NullReferenceException(
-                    "BlockHashes doesn't have sender address.");
-            }
-
-            Peer peer = _peers.Keys.FirstOrDefault(p => p.Address.Equals(from));
-            if (peer == null)
+            if (!(message.Remote is BoundPeer peer))
```

Comment:
```suggestion
            if (message.Remote is UnBoundPeer)
```

Reply:
Or, can `message.Remote` be `null`?

Reply:
Yes, it cannot be null in formal cases, but in case of malicious peer trying to send invalid messages, it can be null.